res/css/views/rooms/_EventTile.scss

```diff
@@ -311,6 +311,7 @@ $left-gutter: 64px;
         .mx_EventTile_line {
             /* ideally should be 100px, but 95px gives us a max thumbnail size of 800x600, which is nice */
             margin-right: 110px;
+            min-height: 2.2rem;
```

Comment:
This should be using one of our `$font` variables given we support font scaling.

Reply:
Can you pls elaborate? I didn't exactly understand what to change. As the msg is blank so their is no need to care abt the font.

Reply:
We don't currently use `rem` in our heights and have the variables for sizing of lines broken out to `$font` variables elsewhere - if we don't use the font variables, we end up in a case where the timestamps can still overlap if the user has a high enough scale value, so we should reserve the space of what *would be* 1 line's worth of text.

Reply:
<img width="323" alt="Screenshot 2022-04-04 at 12 50 38 PM" src="https://user-images.githubusercontent.com/54431564/161493451-b594830e-28d9-437b-b262-dd31f7edab57.png">
Does this work?

Reply:
if 14px is the variable we use normally, yes :D

Reply:
Great. So I will commit this change then. Thanks for your review!

Reply:
A small query, I am getting an error on line 629 although I didn't do any changes to it.
<img width="468" alt="Screenshot 2022-04-04 at 1 00 22 PM" src="https://user-images.githubusercontent.com/54431564/161494916-ce53692e-a550-4dde-a433-a938e670696e.png">
<img width="400" alt="Screenshot 2022-04-04 at 1 00 34 PM" src="https://user-images.githubusercontent.com/54431564/161494924-3fb6c1fa-3354-488d-87b3-7219f7365dd7.png">

Reply:
I think that might be your IDE trying to use a different SCSS markup than we actually use, so it errors. It shouldn't affect the CI on your PR though.

Reply:
I checked the scss documentation and it says that @mixin requires curly braces inside which the properties should be specified. Like the
<img width="217" alt="Screenshot 2022-04-04 at 1 08 59 PM" src="https://user-images.githubusercontent.com/54431564/161496356-372bd571-c257-4f7a-ba70-f1494da11c3b.png">
 one below. So I have added curly braces. Hope it doesn't create a problem.

Reply:
@turt2live Is some other change required?

Reply:
@turt2live Any updates on this PR?
Is any other change required from my side